functions/01_TotalActivation/Temporal_PFM/PFM_Temporal_OneTimeCourse.m

```diff
@@ -11,14 +11,14 @@
 % - ParametersOut: structure containing the parameters of the algorithm
 %
 % Implemented by Eneko Uruñuela, 13.12.2022
-function [x,ParametersOut] = TA_Temporal_OneTimeCourse(y,idx_vox,ParametersIn)
+function [Activity_related,Activity_inducing,Innovation,ParametersOut] = TA_Temporal_OneTimeCourse(y,idx_vox,ParametersIn)
```

Comment:
This way we can later save the Activity_inducing and Innovation vectors of one voxel inside matrices in ``MyTemporal_pfm.m``